src/components/procedure/_remove-rule.mdx

```diff
@@ -6,13 +6,13 @@ To **remove a rule** from the Rule board:
 4. Click **Save**.
 
 :::info
-- Individual rules within rulesets can only be disabled, not deleted. To disable an individual rule, click the <i class="fa-light fa-toggle-large-on"></i> toggle to disable the rule.
+- Individual rules within rulesets can only be disabled, not deleted. To disable an individual rule in **[Rule board](https://semgrep.dev/orgs/-/board)**, click the <i class="fa-light fa-toggle-large-on"></i> toggle to disable the rule.
 - When you remove a rule from the Rule Board, all associated findings on Findings page and Dashboard page are removed also.
 :::
 
 You can also remove a rule on the Findings page, to do so, follow these steps:
 
-1. Go to the [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page.
+1. On the Semgrep App [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page.
```

Comment:
I think this needs a verb. Maybe “Start on the…”?

Reply:
```suggestion
1. Go to the Semgrep App [Findings](https://semgrep.dev/orgs/-/findings?tab=open) page.
```